db/migrate/20210506101257_remove_more_defaults_from_application_forms_table.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveMoreDefaultsFromApplicationFormsTable < ActiveRecord::Migration[6.1]
```

Comment:
Note there is a separate PR for this migration - commit was cherry picked - https://github.com/DFE-Digital/apply-for-teacher-training/pull/4671